### PR TITLE
Ability to refund individual tickets in an order

### PR DESCRIPTION
**Pull request by JapSeyz**
Any order using Stripe can now have individual tickets refunded directly from the order management modal.

### CODE REVIEW

**Comment by ylecuyer** [comment]
Please don't commit code formatting changes, where is the actual change in dc93a17 ?

File: config/services.php [hunk 1]
```diff
@@ -36,5 +36,8 @@
 
     'twitter' => [
         'widget_id' => env('TWITTER_WIDGET_ID'),
+    ],
+    'sparkpost' => [
+        'secret' => env('SPARKPOST_SECRET')
```
Comment (ylecuyer) [inline config/services.php]: I don't think it is related to the refund code

File: app/Http/Controllers/EventAttendeesController.php [hunk 2]
```diff
@@ -662,7 +662,7 @@ public function postCancelAttendee(Request $request, $event_id, $attendee_id)
                 $message->to($attendee->email, $attendee->full_name)
                         ->from(config('attendize.outgoing_email_noreply'), $attendee->event->organiser->name)
                         ->replyTo($attendee->event->organiser->email, $attendee->event->organiser->name)
-                        ->subject('You\'r ticket has been cancelled');
+                        ->subject('Your ticket has been cancelled');
```
Comment (ylecuyer) [inline app/Http/Controllers/EventAttendeesController.php]: Squash with previous commit

File: config/services.php (hunk 1 above)
Comment (JapSeyz) [inline config/services.php]: That's true, it is a general fix for using SparkPost. The TransportManager expects that to be there.

It's a bug that it is missing IMO, and thus is fixed here.

File: app/Http/Controllers/EventAttendeesController.php (hunk 2 above)
Comment (JapSeyz) [inline app/Http/Controllers/EventAttendeesController.php]: I am not extremely experienced with Git, and will look up how to do that in the future.

**Comment by JapSeyz** [comment]
Sorry about the formatting change, I didn't catch that. I auto-format it to PSR-2 when I save via PHPStorm, I must've forgotten to turn it off for Attendize on my laptop.

File: config/services.php (hunk 1 above)
Comment (ylecuyer) [inline config/services.php]: Then it should be in an independent commit

File: app/Http/Controllers/EventAttendeesController.php (hunk 2 above)
Comment (ylecuyer) [inline app/Http/Controllers/EventAttendeesController.php]: You just have to do git rebase -i master and follow onscreen instructions

File: config/services.php (hunk 1 above)
Comment (JapSeyz) [inline config/services.php]: I'll remember that.

**Comment by daveearley** [comment]
Hi @JapSeyz , thanks for this. Could you resolve the conflicts on and then resubmit, some files aren't up to date. Cheers!

**Comment by JapSeyz** [comment]
Of course. I will probably do a bit of reworking, and have it save the ticket price, to make it independent of ticket price changes.

However, this will be in two weeks. I have 7 hours of oral exams in the next ten days, that I have to prepare for.
